.github/workflows/aocc-auto.yml

```diff
@@ -86,12 +86,24 @@ jobs:
           make -j3
         working-directory: ${{ runner.workspace }}/build
 
+      # ph5diff tests are in the tools/tests directory so they will get run
+      # here
       - name: Autotools Run Tests
         env:
           NPROCS: 2
```

Comment:
Should NPROCS be unset here?

Reply:
It could be. It's not necessary.

Reply:
Done. We only use NPROCS in the parallel tests and configure (just in case).

Reply:
Wait, no - I'm putting it back in the serial tests since we run the ph5diff tests there.

Reply:
That seems worth reworking since they aren't serial tests

Reply:
Agree, but I'll save that for a different PR.